wandb/integration/kfp/__init__.py

```diff
@@ -0,0 +1,6 @@
+__all__ = ["wandb_log", "unpatch"]
```

Comment:
*nit but we generally put this at the bottom of the file, suprised the linter didn't complain...

Reply:
We can, but I think it's [PEP8 to place `__all__` at the top](https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names)!

Reply:
Touché!  If the PEP8 says so 👍